libcnb-test/src/test_context.rs

```diff
@@ -199,6 +244,20 @@ impl<'a> TestContext<'a> {
     }
 }
 
+/// Downloaded SBOM files.
```

Comment:
Should these new types be added below the `impl<'a> Drop for TestContext<'a>` block, so the different types and their implementations aren't intermixed?

Reply:
Yeah, makes sense. Fixed in d6e74a6,